storage/connectors/redis/src/test/java/feast/storage/connectors/redis/retriever/RedisClusterOnlineRetrieverTest.java

```diff
@@ -160,46 +180,21 @@ public void shouldReturnResponseWithValuesIfKeysPresent() {
 
   @Test
   public void shouldReturnNullIfKeysNotPresent() {
-    FeatureSetRequest featureSetRequest =
-        FeatureSetRequest.newBuilder()
-            .setSpec(getFeatureSetSpec())
-            .addFeatureReference(
-                FeatureReference.newBuilder().setName("feature1").setProject("project").build())
-            .addFeatureReference(
-                FeatureReference.newBuilder().setName("feature2").setProject("project").build())
-            .build();
-    List<EntityRow> entityRows =
-        ImmutableList.of(
-            EntityRow.newBuilder()
-                .setEntityTimestamp(Timestamp.newBuilder().setSeconds(100))
-                .putFields("entity1", intValue(1))
-                .putFields("entity2", strValue("a"))
-                .build(),
-            EntityRow.newBuilder()
-                .setEntityTimestamp(Timestamp.newBuilder().setSeconds(100))
-                .putFields("entity1", intValue(2))
-                .putFields("entity2", strValue("b"))
-                .build());
+    byte[] serializedKey1 = "serializedKey1".getBytes();
+    byte[] serializedKey2 = "serializedKey2".getBytes();
 
-    List<FeatureRow> featureRows =
-        Lists.newArrayList(
-            FeatureRow.newBuilder()
-                .setEventTimestamp(Timestamp.newBuilder().setSeconds(100))
-                .addAllFields(
-                    Lists.newArrayList(
-                        Field.newBuilder().setValue(intValue(1)).build(),
-                        Field.newBuilder().setValue(intValue(1)).build()))
-                .build());
+    when(serializer.serialize(redisKeys.get(0))).thenReturn(serializedKey1);
```

Comment:
do we need to mock serializer? why not to use actual implementation?

Reply:
I have replaced the mock serializer with actual implementation.